docs/book/src/reference/reference.md

```diff
@@ -31,10 +31,12 @@
   - [completion](completion.md)
   - [Artifacts](artifacts.md)
   - [Platform Support](platform.md)
-  - [Writing controller tests](writing-tests.md)
```

Comment:
Just to make sure - was this removed because there is no longer a file named `writing-tests.md` in the `reference` section?